README.md

```diff
@@ -109,133 +224,129 @@ Primary key fingerprint: 1DC9 1579 9132 3D23 FD37  BAA7 A6DB EAC6 40C5 A14B
+File `bitcoin-node.sh`
 
 ```bash
-   curl -O https://bitcoin.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-osx64.tar.gz
-   tar -zxf bitcoin-0.18.0-osx64.tar.gz
-   sudo mkdir -p /usr/local/bin
-   sudo cp bitcoin-0.18.0/bin/bitcoin* /usr/local/bin/.
-   rm -rf bitcoin-0.18.0*
-   As a validation, you can run *bitcoind -daemon*. Run *bitcoin-cli stop* afterwards.
+#!/bin/bash
+bitcoind -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -debug=net -deprecatedrpc=signrawtransaction -addresstype=legacy -deprecatedrpc=accounts -deprecatedrpc=generate -datadir=<PATH_TO_DATA_DIR>
 ```
 
-Create the scripts below as a file and its content:
+File `btc-regtest.sh`
 
-**bitcoin-node.sh**
 ```bash
 #!/bin/bash
-
-bitcoind -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -debug=net -deprecatedrpc=signrawtransaction -addresstype=legacy -deprecatedrpc=accounts -deprecatedrpc=generate -datadir=PATH/TO/DATA/DIRECTORY
+bitcoin-cli -regtest --rpcuser=rsk --rpcpassword=rsk --rpcport=18332 $@
 ```
-**btc-regtest.sh**
-```bash
-#!/bin/bash
 
-bitcoin-cli -regtest --rpcuser=rsk --rpcpassword=rsk --rpcport=18332 $@
+Start the Bitcoin node with (use another terminal as you will have to keep it running)
+
+```bash
+./scripts/bitcoin-node.sh
 ```
 
-- Run the **bitcoin-node.sh** script to start the Bitcoin node
-- Generate 1 block using the command **./btc-regtest.sh generate 1**
-- Run **./btc-regtest getblockcount** to see the amount of blocks available
+Generate 1 block with
 
+```bash
+ ./scripts/btc-regtest.sh generate 1
+```
 
-### Running Powpeg node using local RSKj source code
+This should return the block hash as an object.
 
-If you need to run the Powpeg node using a local version of RSKj instead of relying on the dependencies been resolved by Maven, you will have to add a customization file.
-Create a file named `DONT-COMMIT-settings.gradle` with the following content:
+Get the amount of blocks available with
 
-```gradle
-includeBuild('<PATH-TO-RSKJ-SOURCE-CODE>') {
-    dependencySubstitution {
-        all { DependencySubstitution dependency ->
-            if (dependency.requested instanceof ModuleComponentSelector
-                    && dependency.requested.group == 'co.rsk'
-                    && dependency.requested.module == 'rskj-core'
-                    && dependency.requested.version.endsWith('SNAPSHOT')) {
-                def targetProject = project(":${dependency.requested.module}")
-                if (targetProject != null) {
-                    println('---- USING LOCAL ' + dependency.requested.displayName +' PROJECT ----')
-                    dependency.useTarget targetProject
-                }
-            }
-        }
-    }
-}
+```bash
+./scripts/btc-regtest getblockcount
 ```
 
-**Note:** Change PATH-TO-RSKJ-SOURCE-CODE value to your local Rskj path.
+This should return an integer.
```

Comment:
We could also add this link as reference https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line